src/SwedbankPay.Sdk.Tests/TestHelpers/TestHelper.cs

```diff
@@ -19,7 +19,7 @@ public static IConfigurationRoot GetIConfigurationRoot(string outputPath)
 
             #elif RELEASE
 
-            var json = $@"{{ ""SwedbankPayConnectionSettings"": {{ ""Token"": ""{Environment.GetEnvironmentVariable("SwedbankPayConnectionSettings.Token", EnvironmentVariableTarget.User)}""}}}}";
+            var json = $@"{{ ""SwedbankPayConnectionSettings"": {{ ""Token"": ""{Environment.GetEnvironmentVariable("MerchantToken", EnvironmentVariableTarget.User)}""}}}}";
```

Comment:
It seems a lot more elegant to do `.AddEnvironmentVariables()` to the `ConfigurationBuilder` than this?